reference/api/keys.md

```diff
@@ -4,20 +4,132 @@ The `/keys` route allows you to create, manage, and delete API keys. To use thes
 
 [Learn more about managing keys and their rights](/learn/security/master_api_keys.md).
 
+## Key object
+
+```json
+{
+  "name": "Default Search API Key",
+  "description": "Use it to search from the frontend code",
+  "key": "0a6e572506c52ab0bd6195921575d23092b7f0c284ab4ac86d12346c33057f99",
+  "uid": "74c9c733-3368-4738-bbe5-1d18a5fecb37",
+  "actions": [
+    "search"
+  ],
+  "indexes": [
+    "*"
+  ],
+  "expiresAt": null,
+  "createdAt": "2021-08-11T10:00:00Z",
+  "updatedAt": "2021-08-11T10:00:00Z"
+}
+```
+
+### `name`
+
+**Type**: String
+**Default value**: `null`
+**Description**: A human-readable name for the key
+
+### `description`
+
+**Type**: String
+**Default value**: `null`
+**Description**: A description for the key. You can add any important information about the key here
+
+### `uid`
+
+**Type**: String
+**Default value**: N/A
+**Description**: A [uuid v4](https://www.sohamkamani.com/uuid-versions-explained) to identify the API key. If not specified, it is automatically generated by Meilisearch
+
+### `key`
+
+**Type**: String
+**Default value**: N/A
+**Description**: An alphanumeric key value generated by Meilisearch by hashing the `uid` and the master key on API key creation. Used for authorization when [making calls to a protected Meilisearch instance](/learn/security/master_api_keys.md#communicating-with-a-protected-instance).
```

Comment:
I'm not sure I like removing the period, but it felt inconsistent with the majority of the other field descriptions. Alternatively we could add periods to all other descriptions, but this solution seems more efficient.

```suggestion
**Description**: An alphanumeric key value generated by Meilisearch by hashing the `uid` and the master key on API key creation. Used for authorization when [making calls to a protected Meilisearch instance](/learn/security/master_api_keys.md#communicating-with-a-protected-instance)
```